### PR TITLE
CIP-0075? | Fair Stakepool Rewards

### PR DESCRIPTION
**Pull request by TobiasFancee**
This CIP makes changes to the current reward scheme to prioritize fairness and increase decentralization. These changes are necessary if we are to achieve a truly egalitarian network. There are two spreadsheets included in this CIP. One for comparing the current scheme to this proposal and another to highlight the problem with current pledge benefit.

Cardano Forum Post: https://forum.cardano.org/t/cip-fair-stakepool-rewards/109368
Current draft of proposal in branch: https://github.com/TobiasFancee/CIPs-1/tree/master/CIP-Fair_Stakepool_Rewards

### CODE REVIEW

**Comment by rphair** [comment]
@TobiasFancee nice to see this begin on the Cardano Forum with such enthusiastic discussion.  I've recommended an upcoming Biweekly CIP Editors' meeting to have a scope of all CIPs with this theme (revisions to PoS rewards & incentives), since a few more of them including yours have come in recently.  If such an agenda is confirmed I'll post it here.

The Discord where we have these meetings online ([invite](https://discord.gg/kyaTyzkBqd)) also has a `#...cip-0050...` channel (I've asked the admin to rename it more generally; hopefully will happen soon) which now includes multiple CIPs in this theme.  I didn't see you there yet & it would be great to have you included in that discussion.  It tends to have more detail & "branstorming" than we ever see on the GitHub thread, and also more focused peer review than on the Cardano Forum. 🤓

File: CIP-Fair_Stakepool_Rewards/README.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+# Backwards Compatibility
+
+This proposal includes parameter changes, one parameter removal, and a change to the rewards calculation. Because of the parameter removal and changes to the rewards calculation, a hardfork will be necessary for implementation.
+
+# Path to Active
+
+This proposal must be approved by both IOG as well as the community before implementation. Community consensus should be a high priority for proposals that modify staking rewards.
+
+Implementation can be staged to reduce shock to the network:
```
Comment (KtorZ) [inline CIP-Fair_Stakepool_Rewards/README.md:168]: I'd encourage to clearly define an implementor in this plan. In particular, if it is to be people working at IOG, they would need to sign off on this plan.

File: CIP-Fair_Stakepool_Rewards/README.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
+---
+CIP: 75?
+Title: Fair Stakepool Rewards 
+Author: Tobias Fancee <tobiasfancee@gmail.com>
+Comments-URI:
+Status: Draft
```
Comment (KtorZ) [inline CIP-Fair_Stakepool_Rewards/README.md:6]:
```suggestion
Status: Proposed
```

File: CIP-Fair_Stakepool_Rewards/README.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
+---
+CIP: 75?
```
Comment (KtorZ) [inline CIP-Fair_Stakepool_Rewards/README.md:2]:
```suggestion
CIP: 75
```

**Comment by KtorZ** [comment]
We discussed this proposal in today's editors meeting (number 57). As for other proposals regarding the reward sharing scheme or the ledger rule, the proposal shall be merged as Proposed (with the above addition) while the founding members work with the community on a framework to review and proceed with changes in those areas of the protocol (see discussion in the -- upcoming -- meeting notes)

**Comment by TobiasFancee** [comment]
Sorry for the late reply. I've made the necessary changes for merging. Let me know if anything needs to be changed.

**Comment by TobiasFancee** [comment]
@KtorZ

**Review by rphair — APPROVED** [review]
thanks @TobiasFancee - this looks ready to go now.  We still have to update according to #389 but I would rather put this through as-is to reflect the work done so far, and then go back later when in a more routine editing mode and correct the header (and maybe some document structure) according to the new CIP-0001 guidelines... I'll add it to that list once I get the notification that this has been merged.

**Comment by rphair** [comment]
Cross-referencing https://github.com/cardano-foundation/CIPs/pull/421 where work on this proposal continues.

**Pull request closed without merging**